jsettlers.common/src/main/resources/jsettlers/common/buildings/big_temple.xml

```diff
@@ -148,9 +148,8 @@
 
 	<door dx="0" dy="2" />
 
-	<constructionStack dx="5" dy="6" material="STONE" buildrequired="8" />
+	<constructionStack dx="5" dy="6" material="STONE" buildrequired="14" />
```

Comment:
I don't think this is a good idea. This would change the balance and feeling of the game quite a bit. Furthermore, you usually don't have so much building materials available that you have to "store" them at construction sites. If you do, you should probably build more...